pkg/registry/core/shoot/strategy.go

```diff
@@ -115,6 +115,10 @@ func mustIncreaseGeneration(oldShoot, newShoot *core.Shoot) bool {
 			if val, ok := common.GetShootOperationAnnotation(newShoot.Annotations); ok && val == common.ShootOperationRetry {
 				mustIncrease = true
 			}
+		case core.LastOperationStatePending:
+			if newShoot.Status.LastOperation.Type == core.LastOperationTypeRestore {
+				mustIncrease = true
+			}
```

Comment:
Why do we need this? Is the `Pending` last operation state ever used in the code? I think we added it but never used it, or?

Reply:
In the end of the migration  we set the shoot status to Restore Pending, in order to trigger generation increase.

Reply:
I see, thanks!